xesar-connect/src/main/kotlin/com/open200/xesar/connect/extension/XesarConnectInstallationPointExt.kt

```diff
@@ -410,3 +410,24 @@ fun XesarConnect.queryStreamInstallationPoint(
 ): Flow<InstallationPoint> {
     return queryStream(InstallationPoint.QUERY_RESOURCE, params, requestConfig)
 }
+
+/**
+ * Configures the Bluetooth state of an installation point asynchronously.
+ *
+ * @param installationPointId The ID of the installation point
+ * @param bluetoothState The Bluetooth state to set
+ * @param requestConfig The request configuration (optional).
+ */
+suspend fun XesarConnect.configureBluetoothStateAsync(
+    installationPointId: UUID,
+    bluetoothState: String,
```

Comment:
Maybe we could use an enum with the values: `ENABLED`, `DISABLED`, `NOT_SUPPORTED`?
If so, we should also update it in the command data class and the InstallationPointChangedEvent.